.github/workflows/canary-aarch64-linux.yml

```diff
@@ -65,14 +65,19 @@ jobs:
           set -x
           DJL_ENGINE=onnxruntime ./gradlew clean run
           rm -rf /root/.djl.ai/
+      - name: Test Huggingface tokenizers
+        working-directory: canary
+        run: |
+          set -x
+          DJL_ENGINE=tokenizers PYTORCH_PRECXX11=true ./gradlew clean run
+          rm -rf $HOME/.djl.ai/
```

Comment:
Do we need to update the above tests to clean the correct cache dir? The above tests are using `/root/...`, but DJL should use the one you have here. I think for github runners, `$HOME` points to `/github/home`

Reply:
good catch. fixed